src/applications/facility-locator/tests/e2e/gaEvents.cypress.spec.js

```diff
@@ -63,6 +63,24 @@ describe('Google Analytics FL Events', () => {
       cy.get('#map-id', { timeout: 10000 }).should(() => {
         assert.isObject(win.dataLayer.find(d => d['fl-map-miles-moved']));
       });
+
+      cy.findByText(/austin va clinic/i, { selector: 'a' })
+        .first()
+        .click();
+
+      cy.get('#facility-detail-id', { timeout: 10000 }).should(() => {
+        assert.isObject(
+          win.dataLayer.find(d => d.event && d.event === 'fl-results-click'),
```

Comment:
What do you think about making some utility functions for these assertions, e.g.

```js
const assertDataLayerEvent = (win, eventName) => {
  assert.isObject(win.dataLayer.find(d => d.event && d.event === eventName));
}
```
... and

```js
const assertDataLayerItem = (win, item) => {
  assert.isObject(win.dataLayer.find(d => d[item]))
}
```

Reply:
done